t/issue/948.t

```diff
@@ -0,0 +1,81 @@
+use strict;
+use warnings;
+
+use Rex::Virtualization;
+use Test::More tests => 6;
+use Data::Dumper;
+use Rex::Helper::Run;
+use Rex::Commands::File;
+use Rex::Commands::Fs;
+use Rex::Commands::Run;
+
+my $image_format = "raw";
+
+sub get_image_format { return $image_format; }
+
+my $count_file = 0;
+my $count_exec = 0;
+
+no warnings 'redefine';
+
+sub Rex::Commands::File::file {
+  my ( $name, %params ) = @_;
+
+  my $fmt = get_image_format();
+  like $params{content}, qr|<driver name="qemu" type="$fmt"/>|,
+    "Found file content for $fmt format.";
+  $count_file++;
+}
+
+sub Rex::Commands::Run::can_run {
+  return 1;
+}
+
+sub Rex::Commands::Fs::unlink {
+  my ($file) = @_;
+}
+
+sub Rex::Helper::Run::i_run {
+  my ($exec) = @_;
+
+  if ( $exec =~ m/virsh.*capabilities/ ) {
+    return eval { local (@ARGV) = ("t/issue/948/capabilities.xml"); <>; };
+  }
+
+  if ( $exec =~ m/^qemu\-img create/ ) {
+    my $fmt = get_image_format();
+    like $exec, qr/^qemu\-img create \-f $fmt/, "qemu-img created a raw image.";
+    $count_exec++;
+  }
+
+  return "";
+}
+
+my $v = Rex::Virtualization->create("LibVirt");
+
+$v->execute(
```

Comment:
Those commands emit normal rex logs mixed together with the TAP output. Can you make the tests quiet, by adding e.g. `$::QUIET = 1;`, please?